bring-it-up/bring-it-up
# M1 fix service card

## Description
#### Summary:
- Fix styling of service card to match the Figma design

#### How to test this PR:
- Step by step of testing the PR
1. Go to App.tsx
2. Create a new Service
3. Use the ServiceCard component and pass the Service you just created as a prop 
4. Run the app and you'll see the service card

#### Notes: 
- Logic for defining tags is not yet figured out (currently uses a hardcoded array to determine tags)
- Number of reviews and link is hardcoded 

## Screenshots:
- Add screenshot(s) of the changes, if applicable
![image](https://user-images.githubusercontent.com/73853606/188577074-9219404d-5899-45d4-85ed-e3f49add6063.png)


## Checklist
